{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"? { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":2,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/authfoundation\/oktaapierror\/errordescription-31v0k"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/OktaAPIError\/errorDescription-31v0k","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Inherited from "},{"type":"codeVoice","code":"LocalizedError.errorDescription"},{"type":"text","text":"."}],"kind":"symbol","metadata":{"role":"symbol","title":"errorDescription","roleHeading":"Instance Property","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"symbolKind":"property","externalID":"s:10Foundation14LocalizedErrorPAAE16errorDescriptionSSSgvp::SYNTHESIZED::s:14AuthFoundation12OktaAPIErrorV","extendedModule":"Foundation","modules":[{"name":"AuthFoundation"}]},"hierarchy":{"paths":[["doc:\/\/AuthFoundation\/documentation\/AuthFoundation","doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/OktaAPIError","doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/OktaAPIError\/LocalizedError-Implementations"]]},"references":{"doc://AuthFoundation/documentation/AuthFoundation/OktaAPIError/errorDescription-31v0k":{"role":"symbol","title":"errorDescription","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[],"identifier":"doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/OktaAPIError\/errorDescription-31v0k","kind":"symbol","type":"topic","url":"\/documentation\/authfoundation\/oktaapierror\/errordescription-31v0k"},"doc://AuthFoundation/documentation/AuthFoundation":{"role":"collection","title":"AuthFoundation","abstract":[{"type":"text","text":"Library that provides foundational features used by Oktaâ€™s authentication libraries, as well as APIs used to work with tokens and user credentials."}],"identifier":"doc:\/\/AuthFoundation\/documentation\/AuthFoundation","kind":"symbol","type":"topic","url":"\/documentation\/authfoundation"},"doc://AuthFoundation/documentation/AuthFoundation/OktaAPIError":{"role":"symbol","title":"OktaAPIError","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"OktaAPIError"}],"abstract":[{"type":"text","text":"Describes errors that may be reported from an Okta API endpoint."}],"identifier":"doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/OktaAPIError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"OktaAPIError"}],"url":"\/documentation\/authfoundation\/oktaapierror"},"doc://AuthFoundation/documentation/AuthFoundation/OktaAPIError/LocalizedError-Implementations":{"role":"collectionGroup","title":"LocalizedError Implementations","abstract":[],"identifier":"doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/OktaAPIError\/LocalizedError-Implementations","kind":"article","type":"topic","url":"\/documentation\/authfoundation\/oktaapierror\/localizederror-implementations"}}}